src/main/resources/locales/en/subscription-management.json

```diff
@@ -18,5 +18,6 @@
   "courtTable": {
     "header1": "Court or tribunal name"
   },
-  "backButton": "Back"
+  "backButton": "Back",
+  "configureListTypeButton": "Select which list types to receive"
```

Comment:
Just a question about the text for the button. This is different to the one suggested in the ticket. I assume you have checked that this is the newly agreed text? About there is the word 'Select' there. Are we implementing some checkboxes or something similar in the later tickets for the users to 'select' something. If not, 'Configure' might be a better word here?

Reply:
Text was suggested by Chris C. Button will link to screen developed in 1510.

Reply:
OK fair enough :)